analyzer-comments/java/salary-calculator/use_ternary_operators.md

```diff
@@ -0,0 +1,3 @@
+# use ternary operators
+
+Consider using `ternary operators` in `%<inMethod>s` to solve this exercise.

```

Comment:
1. Maybe the copy can be a little bit more verbose and explain where and why a ternary operator should be used. As for where, presumably this comment is triggered when the student uses an `if`/`else` statement, right? Perhaps include that in the copy. As for why: maybe we can point out that the goal of this exercise is to learn about ternary operators and so solutions should use them instead of something else.
3. I would not put the phrase "ternary operators" in backticks, as that is usually only done when referring to Java code or keywords.

Reply:
Done!